packages/core/src/base/cacheManagerKeyv.ts

```diff
@@ -167,6 +141,15 @@ export class CacheManagerKeyv implements ICacheManager {
       this.keyv = new Keyv(options.connection, options.keyvOptions);
       this.keyv.on("error", console.error);
     }
+
+    if (this.keyv?.opts?.ttl) {
```

Comment:
Nice null safe navigation...but if this conditional fails what is the TTL? Should there be a warning?

Reply:
Removed log